src/textual/widgets/_tree.py

```diff
@@ -163,6 +163,17 @@ def toggle(self) -> None:
         self._updates += 1
         self._tree._invalidate()
 
+    @property
+    def label(self) -> TextType:
+        """TextType: The label for the node."""
+        return self._label
+
+    @label.setter
+    def label(self, new_label: TextType) -> TextType:
+        """TextType: The label for the node."""
+        self.set_label(new_label)
+        return self.label
```

Comment:
By the way, what does it mean for the setter to return the new value?

Reply:
Actually, come to think of it, in Python, nothing useful at all. Hangover from elsewhere (languages where assignments are also expressions).

Reply:
Ditto Clipper with the likes of Class(y), for those of a particular vintage (if I'm recalling correctly).